src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -177,6 +177,44 @@ void scrollY(double deltaY) {
         orientation.scrollVertically(this, deltaY);
     }
 
+    void scrollXToPixel(double pixel) {
+        orientation.scrollHorizontallyToPixel(this, pixel);
+    }
+
+    void scrollYToPixel(double pixel) {
+        orientation.scrollVerticallyToPixel(this, pixel);
+    }
+
+    Val<Double> totalWidthEstimateProperty() {
+        return orientation.widthEstimateProperty(this);
+    }
+
+    Val<Double> totalHeightEstimateProperty() {
+        return orientation.heightEstimateProperty(this);
+    }
+
+    Val<Double> estimatedWidthPositionProperty() {
+        switch (getContentBias()) {
+            case HORIZONTAL: // vertical flow
+                return breadthPositionEstimateProperty();
+            case VERTICAL: // horizontal flow
+                return lengthPositionEstimateProperty();
+            default:
+                throw new AssertionError("Unreachable code");
+        }
+    }
```

Comment:
Would you mind rewriting this method as

    orientation.horizontalPositionProperty(content)